### PR TITLE
Update punto extension

### PR DESCRIPTION
**Pull request by archibasov**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/punto/src/Dict.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ export const ru_en = new Map([
   ["№", "#"],
   ["%", "$"],
   [":", "%"],
-  [",", "^"],
+  [",", "б"],
```
Comment (red-avtovo) [inline extensions/punto/src/Dict.ts:28]: The former value was correct

File: extensions/punto/src/Dict.ts [hunk 2]
```diff
@@ -161,7 +161,7 @@ export const en_ru = new Map([
   ["b", "и"],
   ["n", "т"],
   ["m", "ь"],
-  [",", "б"],
+  [",", "?"],
```
Comment (red-avtovo) [inline extensions/punto/src/Dict.ts:164]: Same here

File: extensions/punto/src/index.ts [hunk 3]
```diff
@@ -66,11 +73,11 @@ function detectLayout(input: string): string {
 function switchCharacterLayout(char: string): string {
   if (en_ru.has(char)) {
     // console.log(char + " detected in en dict")
-    switchLayout("Russian");
```
Comment (red-avtovo) [inline extensions/punto/src/index.ts:69]: I believe here should be something like `switchLayout(layoutRu.value);`

File: extensions/punto/src/index.ts [hunk 4]
```diff
@@ -66,11 +73,11 @@ function detectLayout(input: string): string {
...
+    // switchLayout("Russian");
     return en_ru.get(char) ?? char;
   } else {
     // console.log(char + " is probably detected in ru dict")
-    switchLayout("ABC");
```
Comment (red-avtovo) [inline extensions/punto/src/index.ts:73]: And here `layoutEn.value`

File: extensions/punto/src/index.ts [hunk 5]
```diff
@@ -66,11 +73,11 @@ function detectLayout(input: string): string {
...
+    // switchLayout("ABC");
     return ru_en.get(char) ?? char;
```
Comment (red-avtovo) [inline extensions/punto/src/index.ts:81]: If you want to support an additional layout, you need to create a separate mapping

**Review by red-avtovo — CHANGES_REQUESTED** [review]

File: extensions/punto/src/index.ts (hunk 3 above)
Comment (archibasov) [reply extensions/punto/src/index.ts:69]: This is a duplicate switch, the main one is already in line 34.

File: extensions/punto/src/Dict.ts (hunk 1 above)
Comment (archibasov) [reply extensions/punto/src/Dict.ts:28]: Yes, perhaps should add setting for a different keyboard.

File: extensions/punto/src/index.ts (hunk 3 above)
Comment (red-avtovo) [reply extensions/punto/src/index.ts:69]: Good call

**Comment by red-avtovo** [comment]
tbh, I would rather create multiple static maps and configure switching between then instead of one dynamic generation of a required map. That would simplify the setup and make it easy to extend

**Pull request closed without merging**